fidesctl/src/fidesctl/core/api_helpers.py

```diff
@@ -67,3 +67,29 @@ def get_server_resource(
     )
 
     return server_resource
+
+
+def list_server_resource(
```

Comment:
`list_server_resource` hits the list endpoint, whereas `get_server_resources` hits the `get` endpoint. You can pass `get_server_resources` a list of keys and it will return them all. For this instance, we just wanted to get _all_ of the resources from an endpoint

Reply:
thank you!